securesystemslib/pyca_crypto_keys.py

```diff
@@ -540,7 +540,7 @@ def create_rsa_encrypted_pem(private_key, passphrase):
 
   encrypted_pem = \
     private_key.private_bytes(encoding=serialization.Encoding.PEM,
-    format=serialization.PrivateFormat.PKCS8,
+    format=serialization.PrivateFormat.TraditionalOpenSSL,
```

Comment:
The docstring for this function should be updated, not only to mention the new PEM format but to correct it. The docstring incorrectly references PyCrypto and 3DES, etc. (this appears to be a copy+paste that wasn't edited).